src/store/allIncidentsSlice/thunks/fetchIncidents.js

```diff
@@ -1,42 +1,54 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
 import { getApprovedIncidents, getFormResponses, getPendingIncidents } from '../../../utils/DashboardHelperFunctions';
+import { sanitizeFormResponse } from './util';
 
 
-/** This thunk fetches all incident data */
+/** This thunk fetches incident data */
 const actionCreator = createAsyncThunk(
-  'allIncidents/fetchAllIncidents',
+  'allIncidents/fetchIncidents',
 
-  /** @param {GetIncidentsPayload} payload */
+  /** @param {FetchIncidentsPayload} payload */
```

Comment:
another formatting change that should match the uniformity of the application.l

Reply:
jsdoc comments like this basically are like taking TypeScript type annotations and putting them in comment so they work in JS. The beauty of it is that VSCode will then give you IntelliSense and autocomplete for anything that uses them.

Because they are in comments, the formatter won't touch them anyway, but I use a consistent format throughout.